ADOPTERS.md

```diff
@@ -60,6 +60,7 @@ to share the details of their usage publicly at this time.
 * [FHE3 GmbH](https://www.fhe3.com/)
```

Comment:
Please update the commit title to be more specific with the name of the project, and also add a commit description of how PITS is using Rook, according to the recommended [commit structure](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).

Reply:
Looks good, except there is a typo in the commit description: `Cerph`